cmd/gandi.go

```diff
@@ -22,7 +22,7 @@ type cli struct {
 	Certificate   certificateCmd   `kong:"cmd,help='Manage Simple Hosting'"`
 	Debug         bool             `kong:"short='d',help='Enable debug logging'"`
 	DryRun        bool             `kong:"help='Enable dry run mode'"`
-	APIKey        string           `kong:"env='GANDI_KEY',help='The Gandi LiveDNS API key (may be stored in the GANDI_KEY environment variable)'"`
+	Token         string           `kong:"env='GANDI_TOKEN',help='The Gandi Personal Access Token (PAT) (may be stored in the GANDI_TOKEN environment variable)'"`
```

Comment:
What about naming it `GANDI_PERSONAL_ACCESS_TOKEN` in order to allow users to easily find what this token is in the Gandi documentation?